src/Configuration/Rector/ArgumentReplacerRecipeFactory.php

```diff
@@ -58,7 +58,7 @@ private function ensureSingleReplacementStrategy(array $data): void
     {
         if (array_key_exists('replacement_value', $data) && isset($data['replacement_map'])) {
```

Comment:
Btw, why 2 different functions for same thing? It's too hard to read


```php
isset($data['replacement_map'], $data['replacement_value'])
```

Reply:
isset returns false if key exist and value is null while array_key_exists returns true.

Reply:
Oh, didn't know that.

Maybe group them into clear method `hasArraysKeys($array, $keys)`?